controllers/che/checluster_controller.go

```diff
@@ -218,7 +218,7 @@ func (r *CheClusterReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	}
 
 	if r.namespace != "" {
-		contollerBuilder.WithEventFilter(util.InNamespaceEventFilter(r.namespace))
+		contollerBuilder = contollerBuilder.WithEventFilter(util.InNamespaceEventFilter(r.namespace))
```

Comment:
should this be `controller` instead of `contoller` ?

Reply:
Yes
BTW, Don't merge this PR yet, I am still testing.